resources/views/website/navbar.blade.php

```diff
@@ -128,15 +132,18 @@
                                            href="{{ route("leaderboards::admin") }}">Leaderboards</a>
                                     @endcan
 
-                                    @if(Auth::user()->hasAllPermissions(['board', 'finadmin']))
+                                    @if(Auth::user()->hasAllPermissions(['board', 'finadmin']) || Auth::user()->can('closeactivities'))
```

Comment:
Maybe make this `hasAllPermissions(['board', 'closeactivities'])` as it is never the intention for anyone but board to have this permission